Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -348,7 +348,8 @@ RefPtr<Element> MarkupAccumulator::replacementElement(const Node& node)
     if (shadowRoot->mode() == ShadowRootMode::UserAgent)
         return nullptr;
 
-    auto element = HTMLTemplateElement::create(HTMLNames::templateTag, node.document());
+    Ref nodeDocument = node.document();
+    auto element = HTMLTemplateElement::create(HTMLNames::templateTag, nodeDocument);
```

Comment:
Can just pass `protectedDocument()` here, no need for a local variable if it is only used once.